docs/docs/troubleshooting/TROUBLESHOOTING.md

```diff
@@ -104,3 +104,16 @@ This guide contains general tips on how to investigate an application deployment
     There are two Terraform locks; one for the infrastructure and another for Terraform state. If you are still experiencing lock issues, change the directory to `./modules/tfstate` and retry the same command.
 
 
+??? tip "How do I deal when remote agents are offline after provision the Bamboo?"
+       
+    **Symptom**
+    
+    The remote agents are installed but remain offline after installation. 
+       
+    **Solution**
+    
+    1. Open the Bamboo application in the browser and login as `administrator`. 
+    2. Go to `Agents` page and select `Agent authentication` tab.
+    3. Select `All` and press `Approve access` button. 
+    4. Wait until all remote agents get online.
```

Comment:
Honestly we didn't see this problem. If you look at the ticket description [here](https://bulldog.internal.atlassian.com/browse/TFKUBE-246) this is reported by `dc-app` team and they was able to make the agents online after install, it means those are not the leftover from imported dataset. 
I added this scenario just in case if the bamboo agents are not online, user know what to do.

Reply:
@nanux you are correct on your comment. A dataset will import with it the agents that were associated with it which will then appear as `offline`